homeassistant/components/foscam/__init__.py

```diff
@@ -30,14 +30,10 @@ async def async_setup_entry(hass: HomeAssistant, entry: FoscamConfigEntry) -> bo
         verbose=False,
     )
     coordinator = FoscamCoordinator(hass, entry, session)
-
+    await hass.async_add_executor_job(session.set_sub_stream_format, 1)
```

Comment:
why do we add this? Would it make sense to add this to `_async_setup` in the coordinator?